shell/platform/darwin/ios/BUILD.gn

```diff
@@ -153,6 +146,146 @@ shared_library("create_flutter_framework_dylib") {
+    "$ocmock_path/OCMock/OCMIndirectReturnValueProvider.m",
+    "$ocmock_path/OCMock/OCMInvocationExpectation.h",
+    "$ocmock_path/OCMock/OCMInvocationExpectation.m",
+    "$ocmock_path/OCMock/OCMInvocationMatcher.h",
+    "$ocmock_path/OCMock/OCMInvocationMatcher.m",
+    "$ocmock_path/OCMock/OCMInvocationStub.h",
+    "$ocmock_path/OCMock/OCMInvocationStub.m",
+    "$ocmock_path/OCMock/OCMLocation.h",
+    "$ocmock_path/OCMock/OCMLocation.m",
+    "$ocmock_path/OCMock/OCMMacroState.h",
+    "$ocmock_path/OCMock/OCMMacroState.m",
+    "$ocmock_path/OCMock/OCMNotificationPoster.h",
+    "$ocmock_path/OCMock/OCMNotificationPoster.m",
+    "$ocmock_path/OCMock/OCMObserverRecorder.h",
+    "$ocmock_path/OCMock/OCMObserverRecorder.m",
+    "$ocmock_path/OCMock/OCMPassByRefSetter.h",
+    "$ocmock_path/OCMock/OCMPassByRefSetter.m",
+    "$ocmock_path/OCMock/OCMRealObjectForwarder.h",
+    "$ocmock_path/OCMock/OCMRealObjectForwarder.m",
+    "$ocmock_path/OCMock/OCMRecorder.h",
+    "$ocmock_path/OCMock/OCMRecorder.m",
+    "$ocmock_path/OCMock/OCMReturnValueProvider.h",
+    "$ocmock_path/OCMock/OCMReturnValueProvider.m",
+    "$ocmock_path/OCMock/OCMStubRecorder.h",
+    "$ocmock_path/OCMock/OCMStubRecorder.m",
+    "$ocmock_path/OCMock/OCMVerifier.h",
+    "$ocmock_path/OCMock/OCMVerifier.m",
+    "$ocmock_path/OCMock/OCMock.h",
+    "$ocmock_path/OCMock/OCMockObject.h",
+    "$ocmock_path/OCMock/OCMockObject.m",
+    "$ocmock_path/OCMock/OCObserverMockObject.h",
+    "$ocmock_path/OCMock/OCObserverMockObject.m",
+    "$ocmock_path/OCMock/OCPartialMockObject.h",
+    "$ocmock_path/OCMock/OCPartialMockObject.m",
+    "$ocmock_path/OCMock/OCProtocolMockObject.h",
+    "$ocmock_path/OCMock/OCProtocolMockObject.m",
+  ]
+  include_dirs = [ "$ocmock_path" ]
+}
+
+ios_test_flutter_path = rebase_path("$root_out_dir/libios_test_flutter.dylib")
+platform_frameworks_path = "$ios_sdk_path/../../Library/Frameworks/"
+
+# NOTE: This currently only support simulator targets because of the install_name.
+# TODO: Switch the install_name and make the test runner copy the dynamic library
+#       into the testing bundle.
+shared_library("ios_test_flutter") {
+  visibility = [ ":*" ]
+  cflags = [
+    "-fvisibility=default",
+    "-F$platform_frameworks_path",
+    "-fobjc-arc",
+    "-mios-simulator-version-min=13.0",
+  ]
+  ldflags = [
+    "-F$platform_frameworks_path",
+    "-Wl,-framework,XCTest",
+    "-Wl,-install_name,$ios_test_flutter_path",
+  ]
+  configs -= [
+    "//build/config/gcc:symbol_visibility_hidden",
+    "//build/config:symbol_visibility_hidden",
+  ]
+  sources = [
+    "framework/Source/FlutterBinaryMessengerRelayTest.mm",
+    "framework/Source/FlutterEngineTest.mm",
+    "framework/Source/FlutterPluginAppLifeCycleDelegateTest.m",
+    "framework/Source/FlutterTextInputPluginTest.m",
+    "framework/Source/FlutterViewControllerTest.mm",
+    "framework/Source/SemanticsObjectTest.mm",
```

Comment:
I wonder if there's anything we can do to make sure people don't forget to add to this when creating new files since it could be easy to miss. We should at least have a testing/ios/IosUnitTests/README.md.

Reply:
added a readme